recipes/libcap/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibcapConan(ConanFile):
+    name = "libcap"
+    license = ("GPL-2.0", "BSD-3-Clause")
```

Comment:
I missed this on previously

Please use `GPL-2.0-or-later` or `GPL-2.0-only` depending on what applies. Since it is hosted on kernel.org I would assume `-only` but I didn't found any clear statement on a quick look. Most of the time people put that info either in the readme or at the top of source files, but this does not seem to be the case here 🤔

Reply:
I don't see any explanations is license `GPL-2.0-or-later` or `GPL-2.0-only` in the repository itself. So it should be `GPL-2.0-only`.
Also in [Debian package](https://metadata.ftp-master.debian.org/changelogs//main/libc/libcap2/libcap2_2.44-1_copyright) the license is referenced as `BSD-3-clause or GPL-2` and [packaging-manuals](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification) explain that they mean `BSD-3-clause or GPL-2.0-only`.